### PR TITLE
added a landing page styled with bootstrap and animated logo

### PR DESCRIPTION
**Pull request by renzosj**
https://github.com/renzosj/goodnight-king/assets/59748996/6d16a61a-ec93-4d13-80e2-aa6ea4080fbb

### CODE REVIEW

File: assets/css/reset.css [hunk 1]
```diff
@@ -11,9 +11,9 @@
   }
   
   /* Gives the body element a parent height to compare against */
-  html {
+  /* html {
```
Comment (sharah24) [inline assets/css/reset.css:14]: Remove trash if you're not using it

**Review by sharah24 — COMMENTED** [review]
Cute I'm liking the color choices

**Comment by sharah24** [comment]
Like visually cute what's your intention cause its not functional and that might not be the right Visual objects to use for buttons and whatever

**Comment by renzosj** [comment]
Just a front-end style template for now. Functionality pending. Still need to finalize main features of project. And you can code any html element as a button with js.

**Pull request merged**